build/webpack-prod-conf.js

```diff
@@ -112,6 +112,13 @@ const webpackConfig = merge(baseWebpackConfig, {
         to: config.build.assetsSubDirectory + '/img',
       }
     ]),
+    // copy opensearch xml file
+new CopyWebpackPlugin([
+  {
+    from: path.resolve(__dirname, '../static/opensearch.xml'),
+    to: config.build.assetsSubDirectory
+  }
+]),
```

Comment:
Just a small detail, but could you fix the identation here to align with the rest?

Reply:
Done